webapp/app.py

```diff
@@ -63,6 +73,80 @@ def _get_examples():
     return examples
 
 
+def _make_github_request(endpoint):
+    # Passing a gh username as a header increases
+    # the API rate limit from 60 to 5000 per hour
+    headers = { "username": random.choice(team_members)["login"] }
+
+    try: 
+        response = requests.get(
+            f'https://api.github.com/{endpoint}',
+            headers,
+            timeout=1
+        )
+
+        response.raise_for_status()
+    except:
+        return {}
+        
+    return response.json()
+
+
+def _get_team_members(contributors):
```

Comment:
I attempted to simplify this a bit, what o you think?
```python3
def _get_team_members(contributors):
    contributors = {contributor["login"]: contributor for contributor in contributors}

    for team_member in team_members:
        member = contributors.get(team_member["login"]) or _make_github_request(f'users/{team_member["login"]}')
        member["role"] = team_member["role"]
        yield member
```

Reply:
Nice!